pkg/router/metrics/metrics.go

```diff
@@ -64,7 +64,7 @@ func (l Listener) authorizeHandler(protected http.Handler) http.Handler {
 		user, ok, err := l.Authenticator.AuthenticateRequest(req)
 		if err != nil {
 			glog.V(3).Infof("Unable to authenticate: %v", err)
-			http.Error(w, "Unable to authenticate due to an error", http.StatusInternalServerError)
+			http.Error(w, "Unable to authenticate due to an error", http.StatusUnauthorized)
```

Comment:
Hrm, this is not correct, because we aren't checking `ok` (which is the normal 401).  What was the actual error?

Reply:
Was it "no permission to check user info"?

Reply:
No, the router was returning 500s when the stats port was hit without username or password.

Reply:
I mean what error was logged in the router.